submissions/krylenger/html-css-popup/index.html

```diff
@@ -0,0 +1,74 @@
+                            <li class="item" tabindex="0"><img src="./icons/003-kitty-45.png"
+                                    alt="cat" /><span>Dance</span></li>
+                            <li class="item" tabindex="0"><img src="./icons/045-kitty-5.png"
+                                    alt="cat" /><span>Coffee?</span></li>
+                            <li class="item" tabindex="0"><img src="./icons/010-kitty-39.png"
+                                    alt="cat" /><span>Tea?</span></li>
+                            <li class="item" tabindex="0"><img src="./icons/011-kitty-38.png"
+                                    alt="cat" /><span>Pirat</span></li>
+                            <li class="item" tabindex="0"><img src="./icons/015-kitty-34.png"
+                                    alt="cat" /><span>Music</span></li>
+                            <li class="item" tabindex="0"><img src="./icons/021-kitty-28.png"
+                                    alt="cat" /><span>iPhone</span></li>
+                            <li class="item" tabindex="0"><img src="./icons/031-kitty-19.png"
+                                    alt="cat" /><span>Hmm..</span></li>
+                            <li class="item" tabindex="0"><img src="./icons/032-kitty-18.png"
+                                    alt="cat" /><span>Gym</span></li>
+                            <li class="item" tabindex="0"><img src="./icons/034-kitty-16.png"
+                                    alt="cat" /><span>Yummmy</span></li>
+                        </ul>
+                        <input class='checkbox toggle-additional-icons' type="checkbox" id="toggle-additional-icons" />
+                        <label for="toggle-additional-icons" class="toggle-additional-icons-label">More</label>
+                        <ul class="pop-up-icons-container-additional">
+                            <li class="item" tabindex="0"><img src="./icons/041-kitty-9.png"
+                                    alt="cat" /><span>Love</span></li>
+                            <li class="item" tabindex="0"><img src="./icons/038-kitty-12.png"
+                                    alt="cat" /><span>Genius</span></li>
+                            <li class="item" tabindex="0"><img src="./icons/036-kitty-14.png"
+                                    alt="cat" /><span>Hero</span></li>
+                            <li class="item" tabindex="0"><img src="./icons/049-kitty-1.png"
+                                    alt="cat" /><span>Party</span></li>
+                            <li class="item" tabindex="0"><img src="./icons/004-idea.png" alt="cat" /><span>Idea</span>
+                            </li>
+                            <li class="item" tabindex="0"><img src="./icons/050-kitty.png" alt="cat" /><span>^_^</span>
+                            </li>
+                            <div class="link-to-kottans"><a href="https://kottans.org/">Even more from Kottans!</a>
+                            </div>
```

Comment:
Element `div` not allowed as a child of element `ul` in this context.

Reply:
✅ done

Reply:
Should I resolve conversation by myself or you should do it after checking adjustments?

Reply:
You must resolve the conversation after pushing your changes.